packs/bestiary-family-ability-glossary/skeleton-frozen.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "defensive",
         "description": {
-            "value": "<p>The skeleton's bones are covered in a thin layer of ice. The skeleton gains immunity to cold and weakness 5 to fire and loses resistance to fire. The skeleton is surrounded by an aura of cold that deals [[/r (max(1,floor(@actor.level/2)))[cold]]]{cold damage equal to half the skeleton's level} to all adjacent creatures at the start of the skeleton's turn (@Check[reflex|dc:|basic] save with a standard DC for its level).</p>"
+            "value": "<p>The skeleton's bones are covered in a thin layer of ice. The skeleton gains immunity to cold and weakness 5 to fire and loses resistance to fire. The skeleton is surrounded by an aura of cold that deals @Damage[(max(1,floor(@actor.level/2)))[cold]] damage to all adjacent creatures at the start of the skeleton's turn (@Check[reflex|dc:|basic] save with a standard DC for its level).</p>"
```

Comment:
`{cold damage equal to half the skeleton's level}`?

Reply:
I removed the flavor text as it ends up getting removed with these PRs during the review. I can re-add it again.